hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/S3ARetryPolicy.java

```diff
@@ -209,6 +210,9 @@ protected Map<Class<? extends Exception>, RetryPolicy> createExceptionMap() {
     // in this map.
     policyMap.put(AWSClientIOException.class, retryAwsClientExceptions);
 
+    // Http Channel issues: treat as communication failure
+    policyMap.put(AwsHttpChannelException.class, connectivityFailure);
```

Comment:
the changes looks good to me. It's just when I commented this line, I was expecting the tests to fail but they don't. 
Because IOE's are always retried.